base_layer/core/src/transactions/transaction_components/transaction_output.rs

```diff
@@ -261,6 +266,7 @@ impl TransactionOutput {
                 .chain(sender_offset_public_key)
                 .chain(commitment)
                 .chain(covenant)
+                //.chain(encrypted_value)
```

Comment:
Should this not be in here?

Reply:
It is, but to add that I have to add `encrypted_value` to the TransactionOutput as well. I'm going to do that in the next PR to keep this one smaller.